README.md

```diff
@@ -1,38 +1,64 @@
 # Slice for Visual Studio Code
 
-This language extension adds support for highlighting Slice files within Visual Studio Code.
-It supports highlighting the following file types:
+This extension provides support for the Slice Interface Definition Language.
+
+## Features
+
+- Syntax highlighting
+- Syntax validation
+- Error detection & reporting
+- 'Go to Definition' jumping
```

Comment:
These two are `.slice` only however.

Reply:
I mention that in their sub-section later on:
`... and is only available for '.slice' files.`

If you prefer it, I can find a way to add it to the main list, but I wanted to keep the bullet points short.

Reply:
That's fine.